portal-impl/src/com/liferay/portal/upgrade/v7_4_x/PortalUpgradeProcessRegistryImpl.java

```diff
@@ -33,6 +33,8 @@ public void registerUpgradeProcesses(
 		upgradeProcesses.put(new Version(9, 0, 0), new UpgradeAddress());
 
 		upgradeProcesses.put(new Version(9, 0, 1), new UpgradeModules());
+
+		upgradeProcesses.put(new Version(9, 0, 2), new UpgradeRegion());
```

Comment:
This should be a minor version increase since we're adding new fields to the database table. See [schema version](https://grow.liferay.com/people?p_p_id=com_liferay_wiki_web_portlet_WikiPortlet&p_p_lifecycle=0&p_p_state=normal&p_p_mode=view&_com_liferay_wiki_web_portlet_WikiPortlet_struts_action=%2Fwiki%2Fview&_com_liferay_wiki_web_portlet_WikiPortlet_redirect=%2Fwelcome%3Fq%3Dversion&_com_liferay_wiki_web_portlet_WikiPortlet_pageResourcePrimKey=850319&p_r_p_http%3A%2F%2Fwww.liferay.com%2Fpublic-render-parameters%2Fwiki_nodeName=Grow&p_r_p_http%3A%2F%2Fwww.liferay.com%2Fpublic-render-parameters%2Fwiki_title=Schema+Version)

And depending on which change goes in first, this will be either 9.1.0 or 9.2.0